api/domain_handlers_test.go

```diff
@@ -98,6 +98,8 @@ func TestAddIneligibleDomain(t *testing.T) {
 	if err != nil {
 		t.Fatalf("[%s] %s", "NewRequest Failed", err)
 	}
+	r.Header.Set("X-Appengine-Cron", "true")
+	r.RemoteAddr = "0.1.0.2"
```

Comment:
nit: make a helper function that returns a properly configured request for this endpoint (instead of repeating this for each occurrence)